docs/configuration/human-readable-byte.md

```diff
@@ -46,10 +46,11 @@ When you have to put a large number for some configuration as above, it is easy
 Given a disk of 1T, the configuration can be
 
 ```properties
-druid.segmentCache.locations=[{"path":"/segment-cache","maxSize":"1t"}]
+druid.segmentCache.locations=[{"path":"/segment-cache-00","maxSize":"1t"},{"path":"/segment-cache-01","maxSize":"1200g"}]
 ```
 
 Note: in above example, both `1t` and `1T` are acceptable since it's case-insensitive.
+Also, only integers are valid as the number part: `1200g` can not be replaced by `1.2t`.
```

Comment:
```suggestion
Also, only integers are valid as the number part. For example, you can't replace `1200g` with `1.2t`.
```

Reply:
Applied this through GitHub's web UI. If necessary, I can squash the commits too. ?